src/pages/NotFound/NotFound.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC } from 'react';
+
+export const NotFound: FC = () => (
+  <>
+    <h1 className="title" style={{ textAlign: 'center' }}>
+      Page not found
+    </h1>
+    <img
+      // eslint-disable-next-line max-len
+      src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExcHdqcmc5eHRxcXA2amg5cDYyNjhrNnJxNDcweHJibGpqNHFvcXJwcyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/VwoJkTfZAUBSU/giphy.gif"
```

Comment:
A cool idea 👍, only if some URL takes up a lot of space, it is better to move this URL outside the component into a separate variable.